### ISSUE TITLE
JavaScript error when trying to focus on a form element in another tab

### ISSUE THREAD

**Issue by muhammad-saleh** [torchbox/wagtail#2934]
Dears,
I upgraded to Wagtail 1.6
And I have some page types with multiple tabs in the admin interface which contains mandatory fields
and now I'm facing an issue when trying to publish a page with an empty mandatory fields in other tabs

Old behavior:
Page do submit and the editing form is opened again with a red circle on the tab with the missing info

In 1.6:
JavaScript error in the console saying that:
```
An invalid form control with name='xxxxxxx' is not focusable.
```
and the page doesn't submit or publish

**Comment by gasman** [comment]
Hi @muhammad-saleh - thanks for the report. What version of Django are you running?

**Comment by muhammad-saleh** [comment]
@gasman Django 1.10

**Comment by m1kola** [comment]
Fixed in #2942

Thank for reporting!
